### PR TITLE
not make PDB configurable

### PR DESCRIPTION
**Pull request by linsun**
Per discussion in https://github.com/istio/istio/issues/11210, due to a current k8s limitation on PDB, we have decided not to make this PDB thing configurable.

### CODE REVIEW

**Comment by sdake** [comment]
@linsun the gate is failing with `Error: release istio failed: PodDisruptionBudget.policy "istio-galley" is invalid: spec: Invalid value: policy.PodDisruptionBudgetSpec{MinAvailable:(*intstr.IntOrString)(0xc427339da0), Selector:(*v1.LabelSelector)(0xc427339dc0), MaxUnavailable:(*intstr.IntOrString)(0xc427339d80)}: minAvailable and maxUnavailable cannot be both set`

File: install/kubernetes/helm/istio/templates/_podDisruptionBudget.tpl [hunk 1]
```diff
@@ -1,8 +1,4 @@
 {{- define "podDisruptionBudget.spec" }}
-{{- if .minAvailable }}
-  minAvailable: {{ .minAvailable }}
+  minAvailable: 1
```
Comment (sdake) [inline install/kubernetes/helm/istio/templates/_podDisruptionBudget.tpl]: It is only possible to set minAvailable or maxAvailable in the PDB, not both.

There is another PR that works to fix this in the queue.

Cheers
-steve

**Comment by sdake** [comment]
The gate failure `FAILED: istio-prometheus-test, run `kubectl logs istio-prometheus-test --namespace istio-system` for more info` is known and I think we are planning to merge a couple fixes for 1.1 to resolve theese problems.  In the meantime you may need to run reteset/friends.

**Comment by linsun** [comment]
@wenchenglu can we get this merged pls? fixes: #11210

**Pull request merged**